.github/actions/release-thread-integration/action.yml

```
@@ -0,0 +1,130 @@
+          echo "No release info artifact found"
+          echo "has_thread_info=false" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Extract deployment message URL
+      id: extract-message-url
+      if: ${{ steps.extract-info.outputs.has_thread_info == 'true' && inputs.slack_response != '' && inputs.skip_thread_post != 'true' }}
+      shell: bash
+      run: |
+        # Extract message details from Slack response
+        slack_response='${{ inputs.slack_response }}'
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)